validator/fielderror.go

```diff
@@ -14,11 +14,19 @@ func (q FieldError) String() string {
 	var sb strings.Builder
 
 	sb.WriteString("Validation failed on field '" + q.Err.Field() + "'")
-	sb.WriteString(", condition: " + q.Err.ActualTag())
 
-	// Print condition parameters, e.g. max=191 -> max { 191 }
-	if q.Err.Param() != "" {
-		sb.WriteString(" " + q.Err.Param())
+	switch q.Err.ActualTag() {
+	case "email":
+		sb.WriteString(", wrong " + q.Err.ActualTag() + " format")
```

Comment:
Here we know that ActualTag() is "email", we don't need to concatenate strings here, just write the whole message: `", wrong email format"`. The same goes for the rest of the messages. Other than that, this is great 👍

Reply:
@FilipNikolovski  this is only for 'email' and 'required' , for 'max' and 'min'  I think it's okay.